astro/src/content/blog/single-sign-on-django-fusionauth.mdx

```diff
@@ -114,7 +114,7 @@ Password: (the default value is fine)
 
 Once the setup is complete, you'll be automatically redirected to the "FusionAuth Setup Wizard", as seen here:
```

Comment:
Not part of this PR, but in other places we just use Setup Wizard in caps with no quotes, should probably do that here, too.

Reply:
fixed.